### ISSUE TITLE
Null-reference spam when launching a vessel

### ISSUE THREAD

**Issue by Nazfib** [krpc/krpc#648]
When launching a vessel, the following error is spammed in the log file:
```
[EXC 00:31:43.891] NullReferenceException: Object reference not set to an instance of an object
	KRPC.SpaceCenter.PilotAddon+ControlInputs.Add (KRPC.SpaceCenter.PilotAddon+ControlInputs other) (at <daba438adcf14fb4a91df2fbef73bb37>:0)
	KRPC.SpaceCenter.PilotAddon.OnFlyByWire (Vessel vessel, FlightCtrlState state) (at <daba438adcf14fb4a91df2fbef73bb37>:0)
	KRPC.SpaceCenter.PilotAddon+<Fly>c__AnonStorey0.<>m__0 (FlightCtrlState s) (at <daba438adcf14fb4a91df2fbef73bb37>:0)
	Vessel.FeedInputFeed () (at <4deecb19beb547f19b1ff89b4c59bd84>:0)
	FlightInputHandler.FixedUpdate () (at <4deecb19beb547f19b1ff89b4c59bd84>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
Neither kRPC nor MechJeb can control the vessel; manual inputs still work.

I had the same with @nullprofile's fork. In that case, I tracked it down to commit e140c0ff521f28df6179f9d443225e74c2e0571d; compiling a version with the custom axis support disabled fixed the problem.

kRPC 0.5.0, KSP 1.12.3 on Linux. The only other mods installed are ModuleManager and MechJeb.

**Comment by djungelorm** [comment]
Looks to be caused by `FlightGlobals.ActiveVessel.FindVesselModuleImplementing<AxisGroupsModule>();` sometimes returning null. A potential fix would be to check if it's null in `KRPC.SpaceCenter.PilotAddon+ControlInputs.Add` and if so ignore the custom axes.

**Comment by djungelorm** [comment]
Could you try with this copy of KRPC.SpaceCenter.dll to see if it fixes it? It's built from the changes in PR #649

https://krpc.s3.amazonaws.com/deploy/fix-648/KRPC.SpaceCenter.dll

**Comment by Nazfib** [comment]
It does indeed fix the problem.
